pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -862,12 +862,24 @@ public String getDescription() {
   }
 
   /**
-   * Returns the value of the connection parameters according to the given {@code Properties}
-   *
+   * Returns the value of the connection parameter from the given {@code Properties} or the
```

Comment:
```suggestion
   * Returns the value of the connection parameter from the given {@link Properties} or the
```

Reply:
so if we change that one we should change all of them in the file